bin/check-redis-ping.rb

```diff
@@ -57,6 +57,12 @@ class RedisPing < Sensu::Plugin::Check::CLI
          long: '--password PASSWORD',
          description: 'Redis Password to connect with'
 
+  option :conn_failure_status,
+         long: '--conn-failure-status EXIT_STATUS',
+         description: 'Exit status for Redis connection failures',
```

Comment:
This could be improved a bit to say "Returns the following exit status for Redis connection failures" which seems a bit more descriptive and does not me to jump to the code to understand what you mean.